bodhi/client/__init__.py

```diff
@@ -379,7 +379,13 @@ def edit(user, password, url, **kwargs):
 @handle_errors
 def query(url, mine=False, **kwargs):
     # User Docs that show in the --help
-    """Query updates on Bodhi."""
+    """Query updates on Bodhi.
+    
```

Comment:
The CI failure seems to be due to whitespace in this line, and on line 385 below.